tests/unittests/tests-nanocoap/tests-nanocoap.c

```diff
@@ -1059,11 +1059,35 @@ static void test_nanocoap__token_length_over_limit(void)
     TEST_ASSERT_EQUAL_INT(8, coap_get_token_len(&pkt));
     TEST_ASSERT_EQUAL_INT(0, pkt.payload_len);
 
-    /* Invalid packet (TKL = 9) */
+    /* Invalid packet (TKL = 15) */
     res = coap_parse(&pkt, buf_invalid, sizeof(buf_invalid));
     TEST_ASSERT_EQUAL_INT(-EBADMSG, res);
 }
 
+/*
+ * Verifies that coap_parse() recognizes token length bigger than allowed.
+ */
+static void test_nanocoap__token_length_ext(void)
```

Comment:
We have four distinct cases to test, right? TKL being:
A:  `0..12`
B: `13`
C: `14`
D: `15`

This test only covers case B.
Either increase the coverage of this test or add three* more tests.

*maybe just one, case A & D are probably already covered somewhere?

Reply:
- A is covered by `test_nanocoap__get_req`  and others
- B is covered by `test_nanocoap__token_length_ext_16`
- C is covered by `test_nanocoap__token_length_ext_269`
- D is covered by `test_nanocoap__token_length_over_limit`